src/Illuminate/Queue/Worker.php

```diff
@@ -254,11 +259,12 @@ protected function daemonShouldRun(WorkerOptions $options, $connectionName, $que
      *
      * @param  \Illuminate\Queue\WorkerOptions  $options
      * @param  int  $lastRestart
+     * @param  string $connectionName
      * @return int|null
      */
-    protected function pauseWorker(WorkerOptions $options, $lastRestart)
+    protected function pauseWorker(WorkerOptions $options, $lastRestart, $connectionName)
```

Comment:
This is a breaking change to the method signature. Please undo this or send the PR to master.